assignments/arrays.js

```diff
@@ -63,34 +63,60 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+console.log(`Car 33 is a` (inventory[32].car_year), (inventory[32].car_make), (inventory[32].car_model));
 
+// how do I properly form this console.log??//
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length -1;
+console.log(inventory[lastCar].car_make + " " + 
+inventory[lastCar.car_model]);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
 
+for (let i=0; i < inventory.length; i++)
```

Comment:
You need braces for the body of the for loop, here you're iterating over the inventory, but then there is no { } where you are specifying what should happen for each item. Afterwards you're sorting the carModels, but since you haven't added anything to it, there is nothing to sort. Please, as you're working on projects, check the output of the console logs. 
```suggestion
for (let i=0; i < inventory.length; i++) {
  carModels.push(inventory[i])
}
carModels.sort()
console.log(carModels)
```